src/test/regress/expected/multi_tenant_isolation_nonblocking.out

```diff
@@ -239,7 +239,8 @@ SELECT isolate_tenant_to_new_shard('orders_streaming', 102, 'CASCADE', shard_tra
 (1 row)
 
 SELECT isolate_tenant_to_new_shard('orders_streaming', 103, 'CASCADE', shard_transfer_mode => 'force_logical');
-ERROR:  multiple shard movements/splits via logical replication in the same transaction is currently not supported
+ERROR:  multiple 'isolate tenant' operations via logical replication in the same transaction is currently not supported
```

Comment:
probably the error has gotten a bit less accurate when it comes to move+split in the same transaction